cdap-docs/admin-manual/source/operations/cdap-ui.rst

```diff
@@ -38,5 +38,234 @@ and are confident of their suitability and compatibility. Earlier versions (and
 suppliers) of components may work, but have not necessarily been either tested or
 confirmed compatible.
 
+UI Customization
+---------------------------------
+Certain properties in the UI can be customized. The ``ui/server/config/themes/default.json`` file contains these properties, along with their default values. As of version 1.0, here is the list of properties that can be customized, and their expected values:
+
+.. list-table::
+   :widths: 15 30 20 30
+   :header-rows: 1
+
+   * - Property Name
+     - Property Attributes
+     - Description
+     - Example
+
+   * - ``brand-primary-color``
+     - No attributes
+     - The brand color to be used throughout UI. Can be in hex/rgb/rgba format.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "brand-primary-color": "#3b78e7"
+
+
+   * - ``navbar-color``
+     - No attributes
+     - The background color of the top nav bar. Can be in hex/rgb/rgba format.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "navbar-color": "rgba(51, 51, 51, 1)"
+
+   * - ``font-family``
+     - No attributes
+     - Comma-separated list of font families to be used in UI, in decreasing order of priority.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "font-family": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif"
+
+   * - ``product-name``
+     - No attributes
+     - The product name to be used in UI. This will displayed on page titles, 'Welcome' modal, and the 'About' modal.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "product-name": "CDAP"
+
+   * - ``product-description``
+     - No attributes
+     - The product description to be used in 'Welcome' modal.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "product-description": "CDAP is an open source framework that simplifies data application development, data integration, and data management."
+
+   * - ``product-logo-navbar``
+     - - ``type``:
+         - ``link``: user is expected to provide a link to the icon image in the ``url`` attribute inside the ``arguments`` map. Path should be from ``ui/cdap_dist`` directory.
+         - ``inline``: user is expected to provide a base64 encoded image in the ``data`` attribute inside the ``arguments`` map.
+       - ``arguments``:
+         - ``url``
```

Comment:
There should be some description for this, and the one below

Reply:
Since you are describing the url and data above, I would only specify this as: arguments for the chosen image type.

Reply:
actually, I think we should completely remove this column. Arguments are only available for these two images right? So let's move the description of the arguments to the Description column, and add a link to the section I pointed to before. Otherwise, there's a lot of wasted space and redundant "No Arguments" Text like below:

![image](https://user-images.githubusercontent.com/3895099/46693785-ab6c6580-cbbf-11e8-85eb-cf0b1bd7f816.png)